src/content/en/tools/chrome-devtools/inspect-styles/edit-dom.md

```diff
@@ -49,6 +49,42 @@ to go into Inspect Element Mode, then click on an element.
 Use the [`inspect`][inspect] method in the console, such as
 `inspect(document.body)`.
 
+### View the rendered and natural sizes of an image {: #image-sizes }
+
+Hover over an `img` tag in the **DOM Tree** to view the rendered and natural sizes of that
+image.
+
+![Rendered and natural image sizes](imgs/image-sizes.png)
+
+### View which image in a source set (srcset) is being used {: #srcset }
+
+The `srcset` property on `img` elements lets you minimize the amount of bytes that you
+must transfer for each image. For example, suppose that you have multiple versions of an image,
+one at 500px width, one at 750px width, and one at 1000px width. If the user's viewport
+is only 400px, then the browser loads the 500px version. If the viewport is 600px, the browser
+loads the 750px version, and so on. If the user widens their viewport, the browser fetches
+each successively larger version of the image as needed. For example, make your viewport narrow
+and load [srcset.salcode.com](http://srcset.salcode.com/), then make it wider and you'll notice
```

Comment:
can you use https://googlesamples.github.io/web-fundamentals/fundamentals/design-and-ux/responsive/media.html instead of http://srcset.salcode.com/? It's a site we own/control, and its served via https

Reply:
I ditched that paragraph. It's not really in keeping with the DevTools docs style to do a long-winded intro of a general web platform feature (but linking off to other docs is OK)

The screenshots are using that link still, is that OK? The section no longer explicitly points users to it. I figure it's a small enough detail not to warrant having to go back and change the screenshots